src/components/Header.jsx

```diff
@@ -0,0 +1,12 @@
+import Link from "next/link";
+import Image from "next/image";
+
+export default function Header() {
+  return (
+    <header className="header">
+      <Link href="/" aria-label="To home page" className="header__link">
+        <Image className="header__logo" src="/sparkbox-logo-text.svg" alt="Sparkbox logo" width='100%' height='100%' />
```

Comment:
`height` and `width` are meant to be unitless on `img` tags, so I'm assuming that's also the case for Next's `Image` component. The `%` might be what caused the issue you mentioned in your other comment. I would change these to match the dimensions of the image in pixels (251 x 68 for the PNG version)

Reply:
The SVG versions of the logo seem a little wonky, did you convert those from the PNG versions? You might be able to grab better version from [the brand assets archive](https://github.com/sparkbox/Brand/releases/tag/archive2022) or even just from sparkbox.com.